tests/Feature/OfficerTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Officer;
+use App\Models\User;
+use Faker\Factory;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class OfficerTest extends TestCase
+{
+    use RefreshDatabase;
+    public function test_only_admin_can_access_officers()
+    {
+        $user = User::factory()->create();
+
```

Comment:
Try with other user roles. You are only testing this on admin. We cannot verify if it cashiers and other roles can access it.